.github/workflows/CI.yml

```diff
@@ -126,3 +126,9 @@ jobs:
           # run the following command to ignore package.json file
           git restore package.json
           git diff --exit-code
+      - name: fantomless
+        run: |
+          # NOTE: maintain fantomless version below in sync with .gitlab-ci.yml
```

Comment:
@realmarv this repo doesn't have a .gitlab-ci.yml file

Reply:
Done